tools/clang/test/HLSLFileCheck/hlsl/workgraph/noinput_rdat.hlsl

```diff
@@ -0,0 +1,371 @@
+// RUN: %dxc -T lib_6_8 %s | %D3DReflect %s | FileCheck %s
```

Comment:
See above about separating mesh nodes from launch types in test files, and mesh nodes should require `lib_6_9`.

Plus, we should add any additional non-mesh node tests to main.

Reply:
Adding the additional tests was the plan. I think there will be others.

Reply:
Separated. I have a raft of tests that will go along to main after I'm sure I have them all.